umbraco-cloud/set-up/project-settings/umbraco-cicd/SampleCiCdPipeline.md

```diff
@@ -0,0 +1,206 @@
+# Getting started with configuring a CI/CD pipeline
+
+Learn how to configure a CI/CD pipeline in Azure DevOps and GitHub Actions Workflows using the scripts provided.
+
+You'll find example shell scripts and pipeline configurations in the "Sample scripts" section, covering both Azure DevOps and GitHub Actions Workflows.
+
+# Why should one configure a sample CI/CD pipeline?
+Umbraco Cloud repositories are not meant to be used as source code repositories. More details on our official documentation.
+
+Once you commit your code to Cloud the build pipeline converts your C# code to DLLs and deploys it on the respective environment.
+
+It is important to note that only C# code is built, and all frontend artifacts need to be built and committed to the repository.
+
+You can use AzureDevops as an external repository and with the pipelines, it will automatically keep your Azure Devops source code repository in sync with the git repository of Umbraco Cloud of the development environment.
+
+![UmbracoCloud CICD sample pipeline](../../images/UmbracoCloudCicdSample.png)
+
+
+# Prerequisites: Setting Up an Umbraco Cloud Project
+
+Before proceeding, you'll need an Umbraco Cloud project, a CI/CD pipeline, and the required files to add to your pipeline for successful interaction with the Umbraco Cloud API.
+
+1. Create an Umbraco Cloud Project: Preferably with a development environment. You can either create a trial,  create a new project, or use an existing one.
+2. Create a new or use an existing CI/CD pipeline: For Azure DevOps see this [page](https://learn.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=azure-devops&tabs=browser). 
+3. Get your set of supporting files: Download sample files as a [zip-file](https://drive.google.com/file/d/1Wpxib2F5-eIyVsSdm3EBvA54eOrgiwLi/view?usp=drive_link). Note, that the only file you properly need to make changes to is the .yaml file. 
+
+**Note**: On the page "What is Umbraco CI/CD Flow", deployments are targeted at the leftmost environment in your Umbraco Cloud setup. This means if you have a Development environment, it will be automatically selected for deployment. If no Development environment exists, the Live environment will be used.
+
+## Obtaining the Project ID and API Key
+
+To get started with API interactions, you'll need to obtain your Project ID and API key. If you haven't already enabled the CI/CD feature, follow these steps:
+
+1. Navigate to the [Umbraco Cloud Portal](https://www.s1.umbraco.io/projects) and select your project.
+2. Go to `Settings` -> `Advanced`. This is where you can generate an API key and find your Project ID.
+
+**Important Note**: The API key is tied to the specific project for which it is generated. Make sure to keep it secure, as it will be used for all subsequent API interactions related to that project.
```

Comment:
```suggestion

{% hint style="info" %}

The API key is tied to the specific project for which it is generated. Make sure to keep it secure, as it will be used for all subsequent API interactions related to that project.

{% endhint %}

```